### DISCUSSION TITLE
Ejecting SuperSlicer deletes the app

### DISCUSSION THREAD

**Discussion by Beach77** [supermerill/SuperSlicer#3558] [Q&A]
I don't know if this is an issue so that's why I'm posting it here. If I eject the SuperSlicer dmg the app stops working so I can't eject it. This has never happened with any other app before so I'm not sure if I'm just being stupid or if it actually is an issue. How do I eject it and have the app keep working? This is on MacBook Air M1.

**Comment by jimmyD30** [comment]
Are you running the app from within the dmg? Make sure you're dragging the app to your applications folder first then open it.

**Reply by Beach77** [reply]
I fixed it by dragging it into Applications in the Finder sidebar. It does not work if I drag it into Applications in the Dock or Launchpad for some reason. Never had this problem before.
